added export and filters to txnotification and claim code.

eoliveros opened the pull request.

eoliveros committed:
added export and filters to txnotification and claim code.

eoliveros committed:
# This is a combination of 3 commits.
# This is the 1st commit message:

added export and filters to txnotification and claim code. Fixed up some of the extra lines and unused codes. Removed the can_{create,delete,edit} in the new view.

# This is the commit message #2:

Enabled csv export for claim code and txnotification. Fixed up some of the extra lines and unused codes.

# This is the commit message #3:

Update models.py

exclude secret for csv export

eoliveros committed:
Enabled csv export for claim code and txnotification. Fixed up some of the extra lines and unused codes.

eoliveros committed:
Update models.py

exclude secret for csv export

djpnewton commented on models.py:
```diff
@@ -9,6 +9,13 @@
 from app_core import app, db
 from utils import generate_key
 
+from flask_admin.babel import lazy_gettext
```
move to other flask_admin imports

eoliveros replied:
can you give me an example of how?

eoliveros replied:
moved this with the other flask_admin something.

djpnewton commented:
there are no authorizer/proposer/viewer roles

eoliveros committed:
Update models.py

rearrange and remove other roles besides admin

djpnewton commented on models.py:
```diff
@@ -1,14 +1,21 @@
 
 from flask import redirect, url_for, request
+from flask_admin.babel import lazy_gettext
+from flask_admin.contrib.sqla import tools
```
not used

eoliveros replied:
removed tis line:
from flask_admin.contrib.sqla import tools

djpnewton commented on models.py:
```diff
@@ -1,14 +1,21 @@
     UserMixin, RoleMixin, login_required, current_user
 from flask_admin.contrib import sqla
 from marshmallow import Schema, fields
+from sqlalchemy.sql import not_, or_
```
not used

eoliveros replied:
removed

djpnewton commented on models.py:
```diff
@@ -1,14 +1,21 @@
 from app_core import app, db
 from utils import generate_key
 
+
```
remove

eoliveros replied:
remove that one extra line

djpnewton replied:
yes

eoliveros committed:
Update models.py

djpnewton commented:
remove create/delete from new model views

eoliveros committed:
Update models.py

Removed create and delete.

djpnewton commented on models.py:
```diff
@@ -9,6 +12,7 @@
 from app_core import app, db
 from utils import generate_key
 
+
```
unnecessary change

eoliveros replied:
removed extra line

eoliveros committed:
Update models.py

djpnewton commented:
can you give better commit notes (and squash into a single commit)

eoliveros committed:
Merge branch 'flask_admin' of https://github.com/eoliveros/zapm_server into flask_admin

eoliveros commented:
I left the:
self.can_edit = True

in there. not sure if thats needed or it can be removed.

djpnewton commented:
no remove the can_edit,

the commits are still not squashed into one commit

eoliveros committed:
Removed the can_edit in the view.

eoliveros committed:
added export and filters to txnotification and claim code. Fixed up some of the extra lines and unused codes. Removed the can_{create,delete,edit} in the new view.

@Enabled csv export for claim code and txnotification. Fixed up some of the extra lines and unused codes.

eoliveros committed:
added export and filters to txnotification and claim code. Fixed up some of the extra lines and unused codes. Removed the can_{create,delete,edit} in the new view.

eoliveros committed:
Merge branch 'flask_admin' of https://github.com/eoliveros/zapm_server into flask_admin

eoliveros committed:
Added csv export and filters to txnotification and claim code. Fixed up some of the extra lines and unused codes. Removed the can_{create,delete,edit} in the new view.

eoliveros committed:
Merge branch 'flask_admin' of https://github.com/eoliveros/zapm_server into flask_admin

djpnewton commented:
see #5
